icu4j/maven-build/maven-icu4j/pom.xml

```diff
@@ -0,0 +1,493 @@
+                      <executable>./file-listing-helper.sh</executable>
+                  </configuration>
+              </execution>
+              <!-- Copy the license file. -->
+              <execution>
+                  <id>copy-license-file</id>
+                  <phase>process-resources</phase>
+                  <goals>
+                      <goal>exec</goal>
+                  </goals>
+                  <configuration>
+                      <executable>cp</executable>
+                      <arguments>
+                        <argument>../../../icu4c/LICENSE</argument>
+                        <argument>${project.build.directory}/classes</argument>
+                      </arguments>
+                  </configuration>
+              </execution>
+          </executions>
+      </plugin>
+
+      <!--
+        Enable configuration of the Manifest file, etc. during the process of packaging the .jar file.
+
+        For future reference: https://stackoverflow.com/questions/38548271/difference-between-maven-plugins-assembly-plugins-jar-plugins-shaded-plugi
+      -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.3.0</version>
+          <configuration>
+            <archive>
+              <manifestEntries>
+
+                <X-Compile-Source-JDK>${maven.compiler.source}</X-Compile-Source-JDK>
+                <X-Compile-Target-JDK>${maven.compiler.target}</X-Compile-Target-JDK>
+
+                <Specification-Title>${jar.spec.title}</Specification-Title>
+                <Specification-Version>${jar.spec.version}</Specification-Version>
+                <Specification-Vendor>${jar.spec.vendor}</Specification-Vendor>
+
+                <Implementation-Title>${jar.impl.title}</Implementation-Title>
+                <Implementation-Version>${jar.impl.version}</Implementation-Version>
+                <Implementation-Vendor>${jar.impl.vendor}</Implementation-Vendor>
+                <Implementation-Vendor-Id>${jar.impl.vendor.id}</Implementation-Vendor-Id>
+
+                <Bundle-ManifestVersion>${jar.manifest.version}</Bundle-ManifestVersion>
+                <Bundle-Name>${jar.name}</Bundle-Name>
+                <Bundle-Description>${jar.description}</Bundle-Description>
+                <Bundle-SymbolicName>${jar.symbolic.name}</Bundle-SymbolicName>
+                <Bundle-Version>${jar.version}</Bundle-Version>
+                <Bundle-Vendor>${jar.vendor}</Bundle-Vendor>
+                <Bundle-Copyright>${jar.copyright.info}</Bundle-Copyright>
+
+                <Main-Class>${jar.main.class}</Main-Class>
+
+              </manifestEntries>
+            </archive>
+          </configuration>
+      </plugin>
+
+    </plugins>
+  </build>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>com.ibm.icu</groupId>
+      <artifactId>icu4j-datafiles</artifactId>
+      <version>73.1-SNAPSHOT</version>
```

Comment:
If the version is the same everywhere (which I think we want), then here you can use
`<version>${project.version}</version>`

Reply:
Done.